contrib/babelfishpg_tsql/src/tsqlIface.cpp

```diff
@@ -180,6 +180,12 @@ template <class T> static void rewrite_geospatial_func_ref_no_arg_query_helper(T
 template <class T> static void rewrite_geospatial_func_ref_args_query_helper(T ctx, TSqlParser::Method_callContext *method, size_t geospatial_start_index);
 template <class T> static void rewrite_function_call_geospatial_func_ref_args(T ctx);
 template <class T> static void rewrite_function_call_geospatial_func_ref_no_arg(T ctx);
+static void handleGeospatialFunctionsInFunctionCall(TSqlParser::Function_callContext *ctx);
+static void handleGeospatialFunctionsInFuncCallExpr(TSqlParser::Func_call_exprContext *ctx);
+static void handleGeospatialFunctionsInLocalIdExpr(TSqlParser::Local_id_exprContext *ctx);
+static void handleGeospatialFunctionsInBracketExpr(TSqlParser::Bracket_exprContext *ctx);
+static void handleGeospatialFunctionsInSubqueryExpr(TSqlParser::Subquery_exprContext *ctx);
+static void handleFullColumnNameCtx(TSqlParser::Full_column_nameContext *ctx);
```

Comment:
The question on reusable code raised by Rob, does it really help by shifting logic in another function?
I dont see a reason why we The current change to introduce new and separate functions is not useful

Reply:
That was exactly what Rob Pointed Out, He also used the same thing
See for example the handling of floating-point constants in exitConstant() ([here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_4_X_DEV/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L2301) and [here](https://github.com/babelfish-for-postgresql/babelfish_extensions/blob/BABEL_4_X_DEV/contrib/babelfishpg_tsql/src/tsqlIface.cpp#L3147)).